docset/windows/international/Copy-UserInternationalSettingsToSystem.md

```diff
@@ -0,0 +1,89 @@
+
+---
+ms.mktglfcycl: manage
+ms.sitesec: library
+ms.author: ertorres
+author: Erik Torres
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: Microsoft.InternationalSettings.Commands.dll-Help.xml
+keywords: powershell, cmdlet
+manager: aandrejs
+ms.date: 10/23/2020
+ms.prod: w10
+ms.technology: powershell-windows
+ms.topic: reference
+online version: 
+schema: 2.0.0
+title: Copy-UserInternationalSettingsToSystem
+ms.reviewer:
+ms.assetid: 
+---
+
+# Copy-UserInternationalSettingsToSystem
+
+## SYNOPSIS
+Copy the current user's international settings (Display language, Input language, Format/locale and Location/GeoID) to one or both of the following:
```

Comment:
I think we should likely add a note that the user must be an admin in order for the operation to succeed. 
For example, on the `Set-WinSystemLocale` cmdlet [here](https://docs.microsoft.com/en-us/powershell/module/international/set-winsystemlocale?view=win10-ps), they use this expression:
> This is a system setting. It can only be changed by a user who has Administrator permissions. Changes take effect after the computer is restarted.

I think it might make sense to use the same wording. :)

Reply:
That's a very good point :D Thanks Jeff :)